<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Semantic similarity</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Word 1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Word 2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sem. sim.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Alpha coef.</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_row_button">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="visualize_checkbox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_row_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete row</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>26</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
